README.md

```diff
@@ -45,6 +45,52 @@ git clone --recursive https://github.com/Apra-Labs/ApraPipes.git
   * Look at the unit_tests/params_test to check for sample usage of parameters in test code
 
 
+## Prerequisites for macOS
+- Update to Xcode 13.2.1 or Recent
+    - Please note you need atleast 40GB of free space on your macPC to update to latest version
+    - Follow this Link to [Install XCode](https://www.freecodecamp.org/news/how-to-download-and-install-xcode/)
+- Install Homebrew
+    ```sh 
```

Comment:
what is the use of "sh" here ? I guess it gives that nice copy to clipboard feature ?

Reply:
Yes it gives a nice border around the command